src/main/java/frc/robot/turret/talonfx/TalonFXTurret.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.turret.talonfx;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.PositionDutyCycle;
+import com.ctre.phoenix6.controls.VelocityDutyCycle;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.constants.GlobalConstants;
+import frc.robot.turret.ITurret;
+import frc.robot.turret.TurretInputsAutoLogged;
+import frc.utils.ctre.CTREDeviceID;
+import frc.utils.devicewrappers.TalonFXWrapper;
+
+public class TalonFXTurret implements ITurret {
+
+    private final TalonFXWrapper motor;
+
+    private final VelocityDutyCycle velocityDutyCycle;
+    private final PositionDutyCycle positionDutyCycle;
+
+    private final StatusSignal<Double> velocitySignal;
+    private final StatusSignal<Double> voltageSignal;
+    private final StatusSignal<Double> currentSignal;
+    private final StatusSignal<Double> positionSignal;
+    private final StatusSignal<Double> accelerationSignal;
```

Comment:
keep it the same order everywhere:) 
(position, velocity, acceleration, current, voltage)